### PR TITLE
imdb api

### PR DESCRIPTION
**Pull request by abhijeet007rocks8**
API added-:
- [IMDb](https://imdb-api.com/api)

Realted to #2

### CODE REVIEW

File: src/content/apis/movies/IMDb.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: IMDb
+description: Access movie and TV information similar to the official one. Get Title, Year, Metascore Rating, IMDB rating, Release date, Runtime, Genre, Directors, Writers, Actors, Plot, Awards, Posters,etc…
+categories:
+  - movies
+features:
+  https: true
+url: https://imdb-api.com/api
+logo: https://seeklogo.com/images/I/imdb-logo-AF81176825-seeklogo.com.jpg
```
Comment (TheLearneer) [inline src/content/apis/movies/IMDb.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/movies/IMDb.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: IMDb
+description: Access movie and TV information similar to the official one. Get Title, Year, Metascore Rating, IMDB rating, Release date, Runtime, Genre, Directors, Writers, Actors, Plot, Awards, Posters,etc…
```
Comment (TheLearneer) [inline src/content/apis/movies/IMDb.md:3]: Please shorten the description

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Thank you @abhijeet007rocks8 for your contribution, but please make sure go through the requested changes for the PR to be merged.

**Comment by abhijeet007rocks8** [comment]
@TheLearneer the Requested changes have been made.

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**